mixins/overlay.html

```diff
@@ -12,7 +12,7 @@
         bottom: var(--lumo-space-m);
         left: var(--lumo-space-m);
         /* Workaround for Edge issue (only on Surface), where an overflowing vaadin-list-box inside vaadin-select-overlay makes the overlay transparent */
-        outline: 0px solid transparent;
+        outline: 0 solid transparent;
```

Comment:
Are you sure this doesn’t create a regression? I’m not certain Edge didn’t care about the unit here…

Reply:
Good point, better play it safe. Added a linter ignore instead.